### ISSUE TITLE
RegisterApiControllers | Filters `IHttpController` with `Controller` suffix

### ISSUE THREAD

**Issue by tenbits** [autofac/Autofac.WebApi#20]
Hello Team,

according to the doc

>  Register types that implement `IHttpController`

but actually it also filters the controllers by the suffix. And we override the contract with 

```
var suffix = typeof(DefaultHttpControllerSelector).GetField(
       "ControllerSuffix", BindingFlags.Static | BindingFlags.Public
);
suffix?.SetValue(null, "Endpoint");
```
So the method doesn't registers our `class FooEndpoint: ApiController`_s_:
https://github.com/autofac/Autofac.WebApi/blob/57ac83fc3a026e1422aac5b71f49719f04f83944/src/Autofac.Integration.WebApi/RegistrationExtensions.cs#L56

Please update the doc, or give the possibility to configurate the `suffix` filter. Thank you!

**Comment by alexmg** [comment]
There is now an overload of `RegisterApiControllers` that allows a custom controller type name suffix to be supplied. This is available in the 4.0.1 package on NuGet.

**Comment by tenbits** [comment]
Thanks, Alex, that's wonderful! Cheers.
